state/initialize.go

```diff
@@ -196,8 +196,6 @@ func Initialize(args InitializeParams) (_ *Controller, err error) {
 			Assert: txn.DocMissing,
 			Insert: &hostedModelCountDoc{},
 		},
-		createSettingsOp(controllersC, ControllerSettingsGlobalKey, args.ControllerConfig),
-		createSettingsOp(globalSettingsC, cloudGlobalKey(args.CloudName), args.ControllerInheritedConfig),
```

Comment:
We should have done this in another PR :wink:

Reply:
I'll rebase. It is in the prior.